TESTS/mbed_hal/uart_bare_metal_test/test_config.h

```diff
@@ -0,0 +1,3 @@
+/* This file has been automatically generated by the python script. */
+/* This file contains serial port baud rate. */
+#define BAUD_RATE 19200
```

Comment:
We could use config for this? to create Config object, add this attribute there and generate config file that would be ignored by git?

In case, this is the best approach, this file possibly should be created by a  test and being ignored in git? Thus not part of this patch?

Reply:
This file is already generated by the script based on provided script parameter. 
I will remove this file from commit.